docs/faqs.md

```diff
@@ -530,3 +530,23 @@ prerequisites for Linux, please look at the VSCode docs.
 https://code.visualstudio.com/docs/remote/linux#_local-linux-prerequisites
 
 </details>
+
+<details style="margin-bottom: 28px;">
```

Comment:
It looks like the closing `details` is missing, probably the browser will auto-close it since it is at the end but it might trip someone up if they are adding a new section.